code/datums/elements/turf_transparency.dm

```diff
@@ -38,6 +41,16 @@
 			return FALSE
 	if(init)
 		our_turf.vis_contents += below_turf
+
+	if(below_turf)
+		var/datum/lighting_object/below_lighting = below_turf.lighting_object
+		var/mutable_appearance/below_turf_without_lighting = new(below_turf)
+
+		if(below_lighting)
+			below_turf_without_lighting.underlays -= below_lighting.current_underlay
```

Comment:
```suggestion
			below_turf_without_lighting.underlays -= below_lighting.current_underlay.appearance
```

might fix things. byond converts all overlay/underlay operations into appearances, and this -= might not be working correctly. 

also, you would have to make sure this doesn't happen if current_underlay had changed but that change hadn't been sent to the underlays

Reply:
Now uses appearance